src/Sulu/Bundle/ContentBundle/Resources/config/serializer/Document.BasePageDocument.xml

```diff
@@ -37,7 +37,7 @@
         <property name="structureType" type="string" groups="preview"/>
         <property name="originalLocale" type="string" groups="preview"/>
         <property name="locale" type="string" groups="preview"/>
-        <property name="path" type="string"/>
+        <property name="path" type="string" exclude="true"/>
```

Comment:
i am no sure but i dont think that this is needed. the only problem i can think of is that the document is not the same after deserialization.